pyatlan/model/packages/asset_export(basic).py

```diff
@@ -0,0 +1,290 @@
+from __future__ import annotations
+
+from typing import List, Optional
+
+from pyatlan.model.enums import CustomWorkflowPackage
+from pyatlan.model.packages.base.custom_package import AbstractCustomPackage
+from pyatlan.model.workflow import WorkflowMetadata
+
+
+class AssetExportPackage(AbstractCustomPackage):
+    """
+    Base configuration for the Asset Export package.
+    """
+
+    _NAME = "asset-export"
+    _PACKAGE_NAME = f"@csa/{_NAME}"
+    _PACKAGE_PREFIX = CustomWorkflowPackage.ASSET_EXPORT.value
+    _PACKAGE_ICON = "http://assets.atlan.com/assets/ph-cloud-arrow-down-light.svg"
+    _PACKAGE_LOGO = "http://assets.atlan.com/assets/ph-cloud-arrow-down-light.svg"
+
+    def __init__(
+        self,
+    ):
+        super().__init__()
+        self._email_addresses = None
+        self._delivery_type = None
+        self._export_scope = None
+        self._parameters = []
+
+    def glossaries_only(
+        self, include_archived: Optional[bool] = None
+    ) -> AssetExportPackage:
+        """
+        Set up the package to export only glossaries.
+        :param include_archived: Whether to include archived glossaries in the export.
+        """
```

Comment:
Let's add a newline between the method docstring sections (`description` and `params/returns/raises`) so that the IDE can render it properly.